#14 customer story

samuelopra opened the pull request:
Fix #14

Test URLs:
- Before: https://main--maestro--hlxsites.hlx.page/
- After: https://customer-story--maestro--hlxsites.hlx.page/

Commit:
customer story section

Commit:
issue-3 customer story intial commit

Commit:
Merge branch 'main' of https://github.com/hlxsites/maestro into customer-story

Commit:
refactor and lint fixes

kunwarsaluja reviewed (commented):
@samuelopra  can you update to use the global vars wherever possible (font-size, weight, spacing etc.) and fetch the latest from the main branch to resolve the conflicts

kunwarsaluja commented on styles/styles.css:
```diff
@@ -318,3 +318,100 @@ main .section {
+  margin: 0;
+}
+
+.section.secondary-banner .default-content-wrapper > p:nth-child(3) {
```
rather than adjusting p:nth-child, would it make sense to adjust the source content for authors such that instead of '**customer story**' as a <p> content, we convert the source content and make it  general that any h3, h4, h5 of the secondary banner would be this specific style

samuelopra replied:
This is done!

Commit:
button container style

Commit:
update button

Commit:
add h selectors instead of p selectors

Commit:
add h selectors instead of p selectors

Commit:
use global styles

kunwarsaluja committed:
Update styles.css

- few adjustments to secondary banner styles
- move blue-background to a separate global style to be reused
